packages/jest-cli/src/__tests__/__snapshots__/generateEmptyCoverage.test.js.snap

```diff
@@ -75,7 +75,7 @@ Object {
       "name": "(anonymous_0)",
     },
   },
-  "path": "/sum.js",
+  "path": "/tmp/sum.js",
```

Comment:
The original unit test does not transpile the code. It does not pass any transformer.

`src code` --> `jest.Runtime.ScriptTransformer` without any transformer --> `code` without any change --> `instrumenter` --> `coverageData`

That's why the coverage data in the snapshot is correct because it just instruments the src code. But in real life, we always use some transformer to process code like jsx, TS.

So what we want is: 
`src code` --> `jest.Runtime.ScriptTransformer` with some transformer --> `code` transpiled --> `instrumenter` --> `coverageData`

However, the `coverageData` is for transpiled code, so we have to source map it to the original code.

Since we use babel, we should use `babel-plugin-istanbul` to instrument the code in order to instrument the ES6 code correctly.

`src code` --> `jest.Runtime.ScriptTransformer` with some transformer and  `babel-plugin-istanbul` --> `code` transpiled & `coverageData`.

Reply:
The reason why I change the `path` which is the file path is that babel require the file path be inside the rootDir. I set the rootDir to `/tmp` and thus the path should be `/tmp/sum.js`